corehq/apps/reports/standard/deployments.py

```diff
@@ -243,9 +288,26 @@ def total_records(self):
 
     @property
     def rows(self):
-        users = self.user_query().run()
-        self._total_records = users.total
-        return self.process_rows(users.hits)
+        if self.warehouse:
+            mobile_user_and_group_slugs = set(
+                # Cater for old ReportConfigs
+                self.request.GET.getlist('location_restricted_mobile_worker') +
+                self.request.GET.getlist(ExpandedMobileWorkerFilter.slug)
+            )
+            users = ExpandedMobileWorkerFilter.user_es_query(
+                self.domain,
+                mobile_user_and_group_slugs,
+                self.request.couch_user,
+            ).values_list('_id', flat=True)
+            sort_clause = self.get_sql_sort()
+            rows = ApplicationStatusFact.objects.filter(user_id__in=users).order_by(sort_clause)
```

Comment:
I think result of `get_sql_sort` is always a string, but should be a list. Once that is fixed, this should say `.order_by(*sort_clause)`